pages/common/sh.md

```diff
@@ -1,20 +1,20 @@
 # sh
 
-> Bourne shell.
-> The standard command language interpreter.
+> Bourne shell, the standard command language interpreter.
+> More information: <https://gnu.org/software/bash/>.
```

Comment:
Why do you link to the Bash manual in the `sh` page. Aren't they two different things?

Reply:
Well I don't even know.  On macOS, `sh` is `/private/var/select/sh` which is a symlink to `/bin/bash`.  On Ubuntu, `/bin/sh` a symlink to `/bin/dash`.  I also couldn't find a good page.

Reply:
I'm worried about the few cases, where `sh` isn't `bash`. Is `sh` even a real shell, or does it always symlink to another shell? As far as I have understood it is just a standardisation concept. 
Linking it to bash probably is correct in most of the cases, but it may be misleading. What about <https://manned.org/sh>?

Reply:
I like it. 👍🏻

Reply:
Done.